.gitignore

```diff
@@ -1 +1,3 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+node_modules/
+package-lock.json
```

Comment:
We don't wanna gitignore package-lock (I nearly fell into this trap cos its a huge file that doesn't _feel_ like it should be committed, but if you have a quick google it will make much more sense!)